docs/user/azure/customization.md

```diff
@@ -17,6 +21,15 @@ The following options are available when using Azure:
 * `type` (optional string): The Azure instance type.
 * `zones` (optional string slice): List of Azure availability zones that can be used (for example, `["1", "2", "3"]`).
 
+## Installing to Existing Networks & Subnetworks
```

Comment:
@jhixson74 can you provide an example of install-config.yaml for byo-vnet.

Reply:
> @jhixson74 can you provide an example of install-config.yaml for byo-vnet.

updated based on this: https://github.com/openshift/installer/pull/2521#issuecomment-547190111